afids_utils/afids.py

```diff
@@ -19,7 +19,7 @@ class AfidPosition:
     x: float = attrs.field()
     y: float = attrs.field()
     z: float = attrs.field()
-    desc: str = attrs.field()
+    desc: str = attrs.field(default="")
```

Comment:
I'd rather not have a default value that's invalid here (even if you want to leave the invalid `desc`s in the test for now)

Reply:
Fair enough, I think alternatively I can load in the valid `descs` and any invalid ones set to "Unknown".